core/tests/protractor_utils/AdminPage.js

```diff
@@ -386,110 +391,135 @@ var AdminPage = function() {
+      CONTRIBUTION_RIGHT_CATEGORY_REVIEW_TRANSLATION,
+      languageDescription);
   };
 
   this.assignVoiceoverReviewer = async function(username, languageDescription) {
-    await _assignReviewer(
-      username, REVIEW_CATEGORY_VOICEOVER, languageDescription);
+    await _assignContributionRights(
+      username,
+      CONTRIBUTION_RIGHT_CATEGORY_REVIEW_VOICEOVER,
+      languageDescription);
   };
 
   this.assignQuestionReviewer = async function(username) {
-    await _assignReviewer(username, REVIEW_CATEGORY_QUESTION);
+    await _assignContributionRights(
+      username, CONTRIBUTION_RIGHT_CATEGORY_REVIEW_QUESTION);
+  };
+
+  this.assignQuestionContributor = async function(username) {
+    await _assignContributionRights(username, CATEGORY_SUBMIT_QUESTION);
   };
 
   this.expectUserToBeTranslationReviewer = async function(
       username, languageDescription) {
-    var reviewRights = await _getUserReviewRightsElement(
-      username, REVIEW_CATEGORY_TRANSLATION);
+    var contributionRights = await _getUserContributionRightsElement(
+      username, CONTRIBUTION_RIGHT_CATEGORY_REVIEW_TRANSLATION);
     var languageList = await Promise.all(
-      reviewRights.map(function(languageElem) {
+      contributionRights.map(function(languageElem) {
         return languageElem.getText();
       }));
     expect(languageList).toContain(languageDescription);
   };
 
   this.expectUserToBeVoiceoverReviewer = async function(
       username, languageDescription) {
-    var reviewRights = await _getUserReviewRightsElement(
-      username, REVIEW_CATEGORY_VOICEOVER);
-    var languageList = await Promise.all(reviewRights.map(
+    var contributionRights = await _getUserContributionRightsElement(
+      username, CONTRIBUTION_RIGHT_CATEGORY_REVIEW_VOICEOVER);
+    var languageList = await Promise.all(contributionRights.map(
       function(languageElem) {
         return languageElem.getText();
       }));
     expect(languageList).toContain(languageDescription);
   };
 
   this.expectUserToBeQuestionReviewer = async function(username) {
-    var reviewRight = await _getUserReviewRightsElement(
-      username, REVIEW_CATEGORY_QUESTION);
+    var contributionRights = await _getUserContributionRightsElement(
+      username, CONTRIBUTION_RIGHT_CATEGORY_REVIEW_QUESTION);
+    await waitFor.visibilityOf(
+      contributionRights, 'Review Right Element taking too long to appear');
+    expect(await contributionRights.getText()).toBe('Allowed');
+  };
+
+  this.expectUserToBeQuestionContributor = async function(username) {
+    var contributionRights = await _getUserContributionRightsElement(
+      username, CATEGORY_SUBMIT_QUESTION);
     await waitFor.visibilityOf(
-      reviewRight, 'Review Right Element taking too long to appear');
-    expect(await reviewRight.getText()).toBe('Allowed');
+      contributionRights, 'Review Right Element taking too long to appear');
+    expect(await contributionRights.getText()).toBe('Allowed');
   };
```

Comment:
Right now the waitFor.visibilityOf calls in these functions have the same error message, which makes debugging hard. Can you make them more descriptive so they uniquely identify the call that failed?